tools/sgphrase/tools.go

```diff
@@ -0,0 +1,41 @@
+package sgphrase
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+
+	"go.einride.tech/sage/sg"
+)
+
+func Run(
```

Comment:
How about calling this `PullTranslations`?

Reply:
Sure.